src/unitxt/catalog/templates/rag/end_to_end/json_predictions.json

```diff
@@ -0,0 +1,8 @@
+{
+    "__type__": "input_output_template",
+    "input_format": "",
+    "output_format": "{{\"answer\": \"{reference_answers}\", \"contexts\" : [\"{reference_contexts}\"],  \"context_ids\" : [\"{reference_context_ids}\"]}}",
+    "postprocessors": [
+        "processors.load_json_predictions"
```

Comment:
@benjaminsznajder - where it this defined in unitxt? I can only see it's used here: 

(fme) yoavkatz@Yoavs-MacBook-Pro unitxt % git grep load_json_predictions

prepare/templates/rag/end_to_end.py:        postprocessors=["processors.load_json_predictions"],
src/unitxt/catalog/templates/rag/end_to_end/json_predictions.json:        "processors.load_json_predictions"

The tests fail because it's not found in the catalog.

Reply:
> @benjaminsznajder - where it this defined in unitxt? I can only see it's used here:
> 
> (fme) yoavkatz@Yoavs-MacBook-Pro unitxt % git grep load_json_predictions
> 
> prepare/templates/rag/end_to_end.py: postprocessors=["processors.load_json_predictions"], src/unitxt/catalog/templates/rag/end_to_end/json_predictions.json: "processors.load_json_predictions"
> 
> The tests fail because it's not found in the catalog.

@yoavkatz  I see that it is a code that was introduced under operator:

from unitxt.operator import SequentialOperator
from unitxt.struct_data_operators import (
    LoadJson,
)

from prepare.tasks.rag.rag_constants import (
    TaskRagEndToEndOutputConstants,
)

from fm_eval.runnables.local_catalogs import add_to_private_catalog

add_to_private_catalog(
    SequentialOperator(
        steps=[
            LoadJson(
                field=f"prediction",
                process_every_value=False,
            ),
        ]
    ),
    "processors.load_json_predictions",
)


I think that this template as a whole is not necessary in unitxt.
don't you think?

Reply:
@yoavkatz I gave an empty template for the card ClapNQ (rag_end_to_end task) 
and remove the templates/rag/end_to_end.py 

is it ok?